src/time-context.js

```diff
@@ -0,0 +1,81 @@
+const MINUTE_IN_MILLIS = 60 * 1000;
+
+const MINUTES = [
+    {key: ['time', 'minute'], value: 'getMinutes'},
+    {key: ['utc', 'time', 'minute'], value: 'getUTCMinutes'}
+];
+const HOURS = [
+    {key: ['time', 'hour'], value: 'getHours'},
+    {key: ['utc', 'time', 'hour'], value: 'getUTCHours'},
+    {key: ['hourOfDay'], value: 'getHours'},
+    {key: ['utc', 'hourOfDay'], value: 'getUTCHours'}
+];
+const DAYS = [
+    {key: ['date', 'day'], value: 'getDate'},
+    {key: ['utc', 'date', 'day'], value: 'getUTCDate'},
+    {key: ['dayOfWeek'], value: 'getDay'},
+    {key: ['utc', 'dayOfWeek'], value: 'getUTCDay'}
+];
+const MONTHS = [
+    {key: ['date', 'month'], value: 'getMonth', transform: function(v) { return v+1; }},
+    {key: ['utc', 'date', 'month'], value: 'getUTCMonth', transform: function(v) { return v+1; }}
```

Comment:
I understand why `transform()` has been added (because "January" => 0, etc.), but is it prudent to deviate from the JS spec just to make it more intuitive for our users?

Reply:
Yeah, because the person building the audience often has no knowledge of or access to the spec.